graphics.tex

```diff
@@ -330,4 +334,62 @@ \subsection*{VIC IV organization and memory access basics}
 	\hline
 \end{tabular}
 
-The table shows which bits must be set to select a video bank. 
+Next the resolution is scaled down (line 40) by setting data bit 7 of the register \$D031 to 0 as mentioned a few
+pages ago.  In order to bring a graphic to the screen a few more settings are required. Within the VIC bank 0
+we have to specify \textbf{where} the data for our graphic is located. In case of hires graphics, the data is divided between the
+bitmap itself and the screen RAM.  In the latter the color of the respective 8x8 block is stored. The following lines
+set these two locations:
+
+\begin{screenoutput}
+50 POKE $D018, PEEK($D018) AND 247 OR 0 :REM BITMAP DATA STARTS AT $0000
```

Comment:
This should use a realistic example of where a program might keep bitmap graphics data. As the text says later, locating bitmap data at $0000 is never a good idea.

I assume that this is trying to avoid a discussion of memory banking and using BANK 128, which leaves no room in bank 0 for bitmap graphics data. It would be better to design a realistic example with an appropriate BANK statement, then refer to the Memory chapter for an explanation.